.github/workflows/benchmark.yml

```diff
@@ -154,4 +154,4 @@ jobs:
           auto-push: ${{ github.event_name == 'push' || github.event_name == 'workflow_dispatch' }}
           save-data-file: ${{ github.event_name == 'push' || github.event_name == 'workflow_dispatch' }}
 
-      
\ No newline at end of file
+      
```

Comment:
bug: we are missing the `TEST_SURREALDB_IMAGE` declaration in the run benchmarks job. I know the code use defaults, but I'd advocate for readability, so this file is self-contained. Please add it here: https://github.com/gofiber/storage/pull/1638/files#diff-aacbd0338bcf598920c120b69adaba2672acc7ea314cfca0f792bebe5fe2acd0R135